PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--- Use a prefix like [TASK], [BUGFIX], [DOC] or [CGL] and provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please try to test the code in multiple browsers and also on a mobile device -->
+
+## Screenshots/links (if appropriate):
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My code follows the code style of this project. (CI will test it anyway and also needs approval)
```

Comment:
For wg-access-server it would be useful to point people towards `go fmt`, Go's built-in formatter. But that's very project-/language-specific, so it doesn't fit here I guess.